pkgs/by-name/st/storcli/package.nix

```diff
@@ -38,15 +39,20 @@ stdenvNoCC.mkDerivation rec {
   # Not needed because the binary is statically linked
   dontFixup = true;
 
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
This would be better to use `passthru.tests` with `testers.testVersion`.

Reply:
Very nice, I didn't know about this.